# DataException.py
#
# Created:  Jan 2015, T. Lukacyzk
# Modified: Feb 2016, T. MacDonald

class Data_Exception(Exception):
    pass